### ISSUE TITLE
aws cloud icons

### ISSUE THREAD

**Issue by vkbytes** [visini/obsidian-icons-plugin#8]
I work a lot on AWS cloud and having their icon set added would be really amazing. They do publish all their architectural icons.
[aws icons](https://aws.amazon.com/architecture/icons/)

**Comment by visini** [comment]
Good idea, this would be useful! Added to potential improvements for future reference.
